sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -1054,5 +1056,329 @@ public virtual Response<SearchSuggestion[]> GetSearchSuggestions(string searchSt
                 throw;
             }
         }
+
+        /// <summary>
+        /// Gets time series insight types in pages asynchronously.
```

Comment:
Might want to confirm with Eric, but I believe he wanted 'time series insights' to be always cased as 'Time Series Insights'